src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -1251,4 +1266,115 @@ public void testPost_200Response_ResponseContentTypeHeaderWithLeadingCharset() {
         assertThat(body.test1, is(equalTo("value1")));
         assertThat(body.test2, is(nullValue()));
     }
+
+    @Test
+    public void testDocumentUpload() {
+        try {
+            FormDataMultiPart multiPart = new FormDataMultiPart();
+            JSONObject jsonObject = new JSONObject();
+            jsonObject.put("type", "LETTER_OF_AUTHORIZATION");
+            jsonObject.put("category", "AUTHORIZATION");
+            List<JSONObject> jsonObjectList = new ArrayList<>();
+            jsonObjectList.add(jsonObject);
+            JSONObject jsonObject1 = new JSONObject();
+            jsonObject1.put("documents", jsonObjectList);
+            BodyPart data =
+                    new FormDataBodyPart(FormDataContentDisposition.name("data").build(), jsonObject1.toString(), MediaType.APPLICATION_JSON_TYPE);
+            multiPart.bodyPart(data);
+
+            Client mockClient = createAndInjectWebResourceClient(hyperwalletApiClient);
+
+            WebResource webResource = mock(WebResource.class);
+            when(mockClient.resource(baseUrl + "/documentUpload")).thenReturn(webResource);
+            WebResource.Builder builder = mock(WebResource.Builder.class);
+            when(webResource.type(MediaType.MULTIPART_FORM_DATA_TYPE)).thenReturn(builder);
+
+            InBoundHeaders headers = new InBoundHeaders();
+            headers.put(HttpHeaders.CONTENT_TYPE, Collections.singletonList("application/json"));
+
+            ClientResponse clientResponse = mock(ClientResponse.class);
+
+            when(builder.put(ClientResponse.class, multiPart)).thenReturn(clientResponse);
+            when(clientResponse.getStatus()).thenReturn(200);
+
+            String hyperwalletUser = "{\n"
+                    + "  \"documents\": [\n"
+                    + "    {\n"
+                    + "      \"category\": \"AUTHORIZATION\",\n"
+                    + "      \"country\": \"CA\",\n"
+                    + "      \"type\": \"LETTER_OF_AUTHORIZATION\",\n"
+                    + "      \"status\": \"NEW\"\n"
+                    + "    }\n"
+                    + "  ]\n"
+                    + "}";
+
+            when(clientResponse.getEntity(String.class)).thenReturn(hyperwalletUser);
+            when(clientResponse.getHeaders()).thenReturn(headers);
+
+            hyperwalletApiClient.put(baseUrl + "/documentUpload", multiPart, HyperwalletUser.class);
+        } catch (Exception exception) {
+        }
```

Comment:
catch block is empty

Reply:
Catch block removed and added assert statements.